test/data/s3/test_s3.py

```diff
@@ -859,15 +860,27 @@ def test_put_one(s3root, objs, expected):
     argnames=["s3root", "blobs", "expected"], **s3_data.pytest_put_blobs_case()
 )
 def test_put_files(tempdir, inject_failure_rate, s3root, blobs, expected):
+    def _write_file_py35(path, data):
+        with open(path, "wb") as f:
+            for d in data.split(b"\n"):
+                f.write(d)
+
+    current_py_version = int(platform.python_version_tuple()[0]) * 10 + int(
```

Comment:
I used to do this but someone recommended against it because there is no guarantee that version numbers are less than 10 so you can have "collisions". Best to use tuples or something like them.